sqlmesh/dbt/context.py

```diff
@@ -185,8 +185,11 @@ def refs(self) -> t.Dict[str, t.Union[ModelConfig, SeedConfig]]:
             for model in t.cast(
                 t.Dict[str, t.Union[ModelConfig, SeedConfig]], {**self._seeds, **self._models}
             ).values():
-                self._refs[model.name] = model
+                self._refs[model.alias or model.name] = model
```

Comment:
I think this change introduce a regression:
`ref` can only be used with the model name AFAIK (I tried to use alias and it fails with dbt).
Moving back to `self._refs[model.name] = model` fixes this issue on my end.

Reply:
What issue are you seeing?